doc/en/user/source/data/webadmin/layergroups.rst

```diff
@@ -102,6 +102,29 @@ Security tab allows to set data access rules at layer group level.
 To create/edit layergroup's data access rules simply check/uncheck checkboxes according to desidered access mode and role. 
 The Grant access to any role checkbox grant each role for each access mode.
 
+Layer Group Styles
+^^^^^^^^^^^^^^^^^^^
+
+The user can also define styles for a Layer Group. By style in this context is meant not an SLD style but different sets of layers/styles that are named and can be thus be referenced through the styles parameter in the various WMS operations (GetMap, GetLegendGraphic, GetFeatureInfo), while the usual Layer Group configuration is kept as the default style.
+
+To add a new style on the :guilabel:`Data` tab scroll down untill reaching the :guilabel:`Layer Group Styles` section and click on :guilabel:`Add new`.
+
+.. figure:: img/add-group-style.png
+
+A form will pop up. Once defined a :guilabel:`name` field, that is mandatory, it is possible to select the list of layer and the corresponding styles, consituting the Layer Group style definition.
+
+.. figure:: img/group-style-definition.png
+
+In the above example the style comprises the same list of the default publishables configuration but different styles have been defined for the ``tiger:giant_polygon`` and ``tiger:poly_landmarks`` layers, respectively ``giant-polygon-2`` and ``poly_landmarks-2``. 
+However the Layer Group Style might also comprise a different list of layers comparing to the default one.
+
+Once a style has been defined it will appear in the ``GetCapabilities`` document and clients will be able to reference it in the ``styles`` parameter of ``GetMap``, ``GetLegendGraphic`` and ``GetFeatureInfo`` operations, using the :guilabel:`name` defined at configuration time.
+
+The usual list of ``publishable`` elements will be treated as the default Style of the Layer Group and used when either no style name is provided or the style name matches the default Layer Group Style name.
```

Comment:
```suggestion
The base configuration will be treated as the default Style of the Layer Group and used when either no style name is provided or the style name matches the default Layer Group Style name.
```
Not sure users have an idea of what a "publishable" is.

Reply:
fixed